src/test/java/de/qabel/desktop/ui/LayoutControllerTest.java

```diff
@@ -1,25 +1,44 @@
 package de.qabel.desktop.ui;
 
+import de.qabel.core.exceptions.QblInvalidCredentials;
 import javafx.scene.Node;
+import org.junit.Before;
 import org.junit.Test;
 
+import java.io.IOException;
+import java.util.Locale;
+
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertTrue;
 
 public class LayoutControllerTest extends AbstractControllerTest {
 
     public static final int MIN_ITEM_COUNT = 2;
 
+    @Override
+    @Before
+    public void setUp() throws Exception {
+        super.setUp();
+    }
+
+    @Test
+    public void testFillQuotaInformation() throws IOException, QblInvalidCredentials {
+        LayoutController controller = createController();
+        assertEquals("30%", controller.quota.getText());
+        assertEquals(30, (int) controller.quotaBar.getMinWidth());
+        assertEquals("667.57 MB free / 953.67 MB", controller.quotaDescription.getText());
```

Comment:
According to the mockup, there should be one digit after the comma. And imho you should add a test that we use the i18n support of mfornos/humanize correctly (just add a test that sets the locale to de_DE and assert for the correct size format)